src/Provider/Geonames.php

```diff
@@ -0,0 +1,269 @@
+<?php
```

Comment:
Why did you put this file here? 
Consider moving it to src/Providers/Geonames/

It may be an issue with you not using the latest master.